Riot/Modules/StartChat/StartChatViewController.m

```diff
@@ -114,8 +117,11 @@ - (void)viewDidLoad
     
     createBarButtonItem = [[UIBarButtonItem alloc] initWithTitle:[VectorL10n start] style:UIBarButtonItemStylePlain target:self action:@selector(onButtonPressed:)];
     self.navigationItem.rightBarButtonItem = createBarButtonItem;
-    
+#ifdef HIDE_MATRIX_ID
```

Comment:
I don't want to add a new macro here, I would prefer to link the placeholder with the property `forceMatrixIdInDisplayName` defined just above
(let see our discussion to link this property with a new BuildSettings in Element)